librosa/util/__init__.py

```diff
@@ -82,4 +82,9 @@
 from . import decorators
 from . import exceptions
 
-__all__ = [_ for _ in dir() if not _.startswith("_")]
+__all__ = ["decorators", "exceptions"]
+__all__ = utils.__all__
+__all__ = files.__all__
+__all__ = matching.__all__
+__all__ = deprecation.__all__
+__all__ = _nnls.__all__
```

Comment:
These bits should have `+=`, if you're going to do it this way, not `=`.  But again, I don't think we need to do all of this if we're not going to block privates.